src/components/select/Select.tsx

```diff
@@ -1,31 +1,60 @@
-import React, { HTMLProps } from 'react';
+import React, { HTMLProps, useContext, useEffect } from 'react';
 import classNames from 'classnames';
 import { FormElementProps } from '../../util/types/FormTypes';
 import FormGroup from '../../util/FormGroup';
+import FormGroupContext from '../formgroup/FormGroupContext';
 
 type SelectProps = HTMLProps<HTMLSelectElement> & FormElementProps;
 
 interface ISelect extends React.FC<SelectProps> {
   Option: React.FC<HTMLProps<HTMLOptionElement>>;
 }
 
-const Select: ISelect = ({ children, ...rest }) => (
-  <FormGroup<SelectProps> inputType="select" {...rest}>
-    {({ className, error, ...restRenderProps }) => (
+const Select: ISelect = props => {
+  const { isInFormGroup, setInputID } = useContext(FormGroupContext);
+  useEffect(() => {
+    if (isInFormGroup && props.id) {
+      setInputID(props.id);
+      return () => {
+        setInputID(undefined);
+      };
+    }
+    return () => {};
+  }, [isInFormGroup, props.id]);
+  if (isInFormGroup) {
+    const { children, className, error, ...rest } = props;
+    return (
       <select
         className={classNames('nhsuk-select', { 'nhsuk-select--error': error }, className)}
-        {...restRenderProps}
+        {...rest}
       >
         {children}
       </select>
-    )}
-  </FormGroup>
-);
+    );
+  }
+  return (
+    <FormGroup<SelectProps> inputType="select" {...props}>
+      {renderProps => <SelectElement {...renderProps} />}
+    </FormGroup>
+  );
+};
 
 const Option: React.FC<HTMLProps<HTMLOptionElement>> = ({ className, ...rest }) => (
   <option {...rest} />
 );
 
+const SelectElement: React.FC<SelectProps> = ({ children, ...props }) => {
```

Comment:
It would probably be easier to just use the standard `<select>` component in the Select component rather than redeclaring it, as it's only used for the final return in the FormGroup